packages/supi/test/storeAdd.ts

```diff
@@ -0,0 +1,26 @@
+import loadJsonFile = require('load-json-file')
+import path = require('path')
+
+import {
+  installPkgs,
+  storeAdd,
+} from 'supi'
+import tape = require('tape')
+import promisifyTape from 'tape-promise'
+import {prepare, testDefaults} from './utils'
+
+const test = promisifyTape(tape)
+
+test('add packages to the store', async (t: tape.Test) => {
+  const project = prepare(t)
+
+  const opts = await testDefaults()
+  // this is needed to initialize the store
+  await installPkgs(['is-negative@2.1.0'], opts)
+
+  await storeAdd(['express@4.16.3'], opts)
+  await project.storeHas('express', '4.16.3')
+
+  const storeIndex = await loadJsonFile(path.join(opts.store, 'store.json'))
+  t.deepEqual(storeIndex['localhost+4873/express/4.16.3'], [])
```

Comment:
You can make a deepEqual of the whole file because it should be empty

```typescript
t.deepEqual(storeIndex, {
  'localhost+4873/express/4.16.3': [],
})
```

Reply:
it also contains another package, I needed to install it to initialize the store in the test

Reply:
oh, I see. But that's an issue, isn't it? I think `pnpm store add` should be able to create a new store

Reply:
It only happens in this test, otherwise it creates the store correctly

Reply:
are you sure? When you run it manually, it probably uses your global store that already exists at `~/.pnpm-store` but tests use a new store in every test case. So to test it manually, you'd need something like `rm -rf store && pnpm store add typescript@1 --store store`

Reply:
That's exactly how I tested it